src/java.net.http/share/classes/jdk/internal/net/http/HttpClientBuilderImpl.java

```diff
@@ -46,7 +46,6 @@ public class HttpClientBuilderImpl implements HttpClient.Builder {
     Authenticator authenticator;
     HttpClient.Version version;
     Executor executor;
-    // Security parameters
     SSLContext sslContext;
```

Comment:
Yes - I toyed with replacing Security with SSL, but decided to remove it instead because there's other field declarations following that have nothing to do with SSL, and this comment doesn't explain anything: it's obvious that SSLContext and SSLParameters are for SSL... Plus, it's the only comment in the whole field declaration list...